pkg/oplabels/labels.go

```diff
@@ -27,6 +27,12 @@ func AddKubernetesLabels(m map[string]string, dc *api.CassandraDatacenter) {
 
 	instanceName := fmt.Sprintf("cassandra-%s", dc.Spec.ClusterName)
 	m[InstanceLabel] = instanceName
+
+	if len(dc.Spec.AdditionalLabels) != 0 {
```

Comment:
It feels a bit odd to add the labels here considering this function is specifically adding well-known kubernetes labels.

Reply:
No, technically this is where I would add them also. I'd make a small change though. Rename this function (refactoring is easiest way to get it everywhere) and then make those kubernetes labels as one step. I don't want to litter the code everywhere with billion different "add this label, add that label" when we could do them all here. It makes the code easier to read when resources are discussed.

Also, this method is called everywhere we currently create resources, so it hits the correct spots.

Reply:
I agree with @burmanm, Will be easier to add other stuff and refactor the method if needed. Since the pattern, when creating an object is to call that method to put the labels.

I changed the name of the method to `AddOperatorLabels`.